auth/token-managers/iam-token-manager.ts

```diff
@@ -35,12 +35,14 @@ function onlyOne(a: any, b: any): boolean {
 }
 
 const CLIENT_ID_SECRET_WARNING = 'Warning: Client ID and Secret must BOTH be given, or the header will not be included.';
+const SCOPE = 'scope';
```

Comment:
I don't know that a constant here is really necessary but I also don't have a problem with it. Up to you but I thought I would at least say that. If you find it helpful, I'm fine with it